media-sound/sinus/sinus-0.1.3.recipe

```diff
@@ -0,0 +1,39 @@
+SUMMARY="Command line tool that creates sine waves"
+DESCRIPTION="Sinus is a little command line tool that creates sine waves, \
+plays them back, and writes then to (.wav) files. It takes a list of frequencies \
+and durations then creates a wave of those frequencies and durations. "
+HOMEPAGE="https://ftp.gwdg.de/pub/linux/misc/blinux/sinus-0.1.3.tar.gz"
+COPYRIGHT="Angus Mackay"
+LICENSE="Public Domain"
+REVISION="1"
+SOURCE_URI=" https://ftp.gwdg.de/pub/linux/misc/blinux/sinus-0.1.3.tar.gz"
+CHECKSUM_SHA256="b81961bbc9854efaf3973701814aefca0fa5ec142c3dd1facac5f4eb08816d35"
+
+ARCHITECTURES="x86_gcc2 x86_64"
+
+PROVIDES="
+	sinus = $portVersion
+	cmd:sinus = $portVersion
+	"
+REQUIRES="
+	haiku
+	"
+
+BUILD_REQUIRES="
+	haiku_devel
+	"
+BUILD_PREREQUIRES="
+	cmd:g++
+	cmd:make
+	"
+
+BUILD()
+{
+	 ./configure
+   make $jobArgs
+}
+
+INSTALL()
+{
+	make install
```

Comment:
Did you fully test this? I'd imagine without the `runConfigure` that you got a lot of policy errors...

Reply:
I did, and that was why I thought I was suppose to only put `./configure`.
How would I fix these errors?

Reply:
Error message I get when I use runConfigure:
```
configure: error: --docdir=/packages/sinus-0.1.3-1/.self/documentation/packages/
sinus: invalid option; use --help to show usage
```